src/vs/platform/extensionManagement/common/extensionGalleryService.ts

```diff
@@ -1310,6 +1311,7 @@ abstract class AbstractExtensionGalleryService implements IExtensionGalleryServi
 		const malicious: IExtensionIdentifier[] = [];
 		const deprecated: IStringDictionary<IDeprecationInfo> = {};
 		const search: ISearchPrefferedResults[] = [];
+		const extensionsEnabledWithPreRelease: string[] = ['sandy081.998-test-incompatible'];
```

Comment:
@sandy081 Is this wanted?

Reply:
Good catch. Added it for testing and forgot to remove.

Thanks